api/s2n.h

```diff
@@ -538,6 +570,19 @@ extern int s2n_rand_set_callbacks(s2n_rand_init_callback rand_init_callback, s2n
 
 /**
  * TLS extensions supported by s2n-tls
+ *
+ * At this time the following extensions are supported:
+ * 
+ * `S2N_EXTENSION_OCSP_STAPLING` - If a client requests the OCSP status of the server
+ * certificate, this is the response used in the CertificateStatus handshake
+ * message.
+ * 
+ * `S2N_EXTENSION_CERTIFICATE_TRANSPARENCY` - If a client supports receiving SCTs
+ * via the TLS extension (section 3.3.1 of RFC6962) this data is returned within
+ * the extension response during the handshake.  The format of this data is the
+ * SignedCertificateTimestampList structure defined in that document.  See
+ * http://www.certificate-transparency.org/ for more information about Certificate
+ * Transparency.
```

Comment:
We support many more extensions. The enum includes others. Why call out these two?